gslib/tests/test_ls.py

```diff
@@ -155,6 +156,21 @@ def test_one_object_with_L_storage_class_update(self):
         stor_update_time,
         datetime.strftime(new_update_time, '%a, %d %b %Y %H:%M:%S GMT'))
 
+  @mock.patch.object(ls.LsCommand, 'WildcardIterator')
+  def test_object_and_prefix_same_name(self, mock_wildcard):
```

Comment:
Maybe use a more descriptive name for this test

Reply:
Oops. I copy pasted another test but forgot to change the name. Thanks for catching this.